www/blog/2021/02/11/megacities/index.md

```diff
@@ -0,0 +1,89 @@
+
+Some examples of these megacities, displayed in Who’s On First’s Spelunker tool:
+
+- [London](https://spelunker.whosonfirst.org/id/101750367/)
+- [Los Angeles](https://spelunker.whosonfirst.org/id/85923517/)
+- [Lima](https://spelunker.whosonfirst.org/id/421186805/)
+- [Chennai](https://spelunker.whosonfirst.org/id/102029537/)
+- [Taipei](https://spelunker.whosonfirst.org/id/102026641/)
+
+## What was updated?
+
+In total, Who’s On First now has 590 locality records that maintain a `wof:megacity` property.
+
+To begin this work, the Populated Places file from Natural Earth was used to identify locality records in Who's On First that should be flagged with a `wof:megacity` property. Once identified, we confirmed the presence or added the `wof:megacity` property on these records maintained in the [`whosonfirst-data`](https://github.com/whosonfirst-data) administrative repositories.
+
+Once these populated places were matched to current Who's On First records, a simple Python script was used to automate the import of the Populated Places’ attributes into new properties on each Who’s On First records. These properties include:
+
+- Name translations
+- Label centroid locations
+- Concordances to Natural Earth and Wikidata, when available
+
+[Here](https://github.com/whosonfirst-data/whosonfirst-data-admin-ir/pull/11/files) is an example of these updates in a GitHub pull request for records in Iran.
+
+#### Updating Geometries
+
+In cases where Who's On First only maintained a `Point` geometry for one of these populated places, we were interested in creating a new `Polygon` geometry. Because many of these localities are represented at more than one placetype in Who's On First - Toronto, for example, is represented as both a [`county`](https://spelunker.whosonfirst.org/id/890457465/), and [`locality`](https://spelunker.whosonfirst.org/id/101735835/) - we were able to use existing "parent" geometries of some of these localities to create (or re-use as) new megacity geometries. In other cases, we hand-curated new geometries using QGIS software.
+
+Because megacities often spread over a larger metropolitan area, we are creating polygons for the “core” cities (which can sometimes be a handful of cities), but we’re exploring creating metropolitan area polygons, too. Please reach out to us if you're interested in those.
```

Comment:
Please reach out to us should have a link to something, even if it's just the `whosonfirst-data` issues page.

Reply:
Resolved in 9c1c7e4